daffodil-lib/src/main/scala/org/apache/daffodil/xml/QNameBase.scala

```diff
@@ -107,10 +108,47 @@ import org.apache.daffodil.exceptions.Assert
  */
 object QName {
 
+  /**
+   * Construct a RefQName from a QName string, and a scope.
+   * @param qnameString
```

Comment:
Remove @param and friends. 
I hate these redundant things. They're always obsolete relative to the code.